res/_redirects

```diff
@@ -0,0 +1,2 @@
+# Redirect everything to index.html, and allow the URL handling to decide what to show.
+/*    /index.html   200
```

Comment:
Does this only apply to URLs that don't match a file?

Reply:
Yes, as far as I can tell: https://perf-html.netlify.com/bb8bc628ef07d98a0856.bundle.js

Reply:
The opposite seems to happen for me:

```
$ curl https://perf-html.netlify.com/bb8bc628ef07d98a0856.bundle.js
<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="initial-scale=1" />
<title>perf.html</title>
<script src='/analytics.js'></script>
<link rel="shortcut icon" href="/favicon.png"></head>
<body style='background-color: #363959; /* ink-70 */'>
<div id="root"></div>
<script type="text/javascript" src="/143310cd260744f8972f.bundle.js"></script></body>
```

It's possible that the service worker affects this in confusing ways, too.

Reply:
@mstange, note the bundle name changed. That's why you got redirected. If you request _that_ URL, then you properly get the JS file.

see https://www.netlify.com/docs/redirects/#note-on-shadowing about this, it seems to do the right thing.

Reply:
I believe this is working as expected. My bundle link was outdated.